silkworm/rpc/http/connection.cpp

```diff
@@ -190,10 +192,11 @@ Task<void> Connection::handle_actual_request(const boost::beast::http::request<b
     vary_ = req[boost::beast::http::field::vary];
     origin_ = req[boost::beast::http::field::origin];
     method_ = req.method();
+    auto encoding = req[boost::beast::http::field::accept_encoding];
 
     auto rsp_content = co_await request_handler_.handle(req.body());
     if (rsp_content) {
```

Comment:
The definition of local variable `encoding` should be moved inside this code branch for better readability